lib/kafka/consumer.rb

```diff
@@ -400,8 +400,10 @@ def fetch_batches(min_bytes:, max_wait_time:)
 
       operation.execute
     rescue NoPartitionsToFetchFrom
-      @logger.warn "There are no partitions to fetch from, sleeping for #{max_wait_time}s"
-      sleep max_wait_time
+      backoff = max_wait_time > 0 ? max_wait_time : 1
+
+      @logger.warn "There are no partitions to fetch from, sleeping for #{backoff}s"
```

Comment:
wouldn't it be better if this was info? I mean having redundant processes isn't something bad and it can be by design, not by accident

Reply:
Hmm. I guess it sort of depends on your use case – I imagine most people will use more than 1 partition.

Reply:
@dasch I know that, you know that but you would be surprised :D the 1 to 1 is pretty common especially at the beginning

Reply:
Sure, I can make it `info`...

Reply:
Thank you :heart: :)